src/energyplus/GeometryTranslator.cpp

```diff
@@ -423,6 +461,7 @@ namespace energyplus {
 
       // get vertices
       Point3dVector vertices = verticesForAzimuthTiltXYZLengthWidthOrHeight(*azimuth, *tilt, *x0, *y0, *z0, *length, *height);
+      vertices = t * vertices;
```

Comment:
That's the actual fix. The building transformation/ building inverse transformation wasn't applied.

(See lines 441 to 446, I thought the compiler would have caught this via `-Wunused-variable` on the existing code but apparently copy-assigning to `t` inside the if block is enough to make it "used")